src/resmom/requests.c

```diff
@@ -4576,19 +4556,16 @@ post_restart(job *pjob, int ev)
 			}
 		}
 
-		new_stime = time_now - (ckpt_time - pjob->ji_qs.ji_stime);
```

Comment:
This snippet of code seemed to have disappeared. It appears that it's accounting for the checkpoint time (ckpt_time)...